content/blog/2020/08/2020-08-04-cdf-graduation.adoc

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: "Jenkins graduates in the Continuous Delivery Foundation"
+tags:
+- announcement
+- cdf
+- general
+- community
+author: oleg_nenashev
+description: >
+  The Jenkins project has graduated in the Continuous Delivery Foundation (CDF).
+  In this article, we discuss what it means to the Jenkins community and what are the future steps for the project.
+opengraph:
+  image: /images/post-images/2020/08-cdf-graduation/opengraph.png
+---
+
+We are happy to announce that the Jenkins project has achieved the **graduated status** in
+the link:https://cd.foundation/[Continuous Delivery Foundation (CDF)].
+This status is official effective Aug 03, 2020.
+Jenkins is the first project to graduate in the CD Foundation.
+Thanks to all contributors who made our graduation possible!
+
+In this article, we will discuss what the CD Foundation membership and graduation mean to the Jenkins community.
+We will also talk about what changed in Jenkins as a part of the graduation,
+and what are the future steps for the project.
+
+NOTE: To know more about the Jenkins graduation,
+see also the link:https://cd.foundation/announcement/2020/08/04/cd-foundation-announces-jenkins-graduation/[announcement on the CDF website].
+
+== How does CDF membership help us?
+
+About 18 months ago, Jenkins link:/blog/2019/03/12/cdf-launch/[became one of the CDF founding projects], along with Jenkins X, Spinnaker and Tekton.
+A new foundation was formed to provide a vendor-neutral home for open source projects used for Continuous Delivery and Continuous Integration.
+Special interest groups were started to foster collaboration between projects and end user companies,
+most notably 
+link:https://github.com/cdfoundation/sig-interoperability[Interoperability],
+link:https://github.com/cdfoundation/sig-mlops[MLOps] and
+link:https://github.com/cdfoundation/sig-security[Security] SIGs.
+Also, a link:https://cd.foundation/ambassador-program-overview-application/[Community Ambassador] role was created to organize local meetups and to provide public-facing community representatives.
+Many former Jenkins Ambassadors and other contributors are now CDF Ambassadors, and they promote Jenkins and other CDF projects there.
+
+Thanks to this membership we addressed key link:/projects/infrastructure/[project infrastructure] needs.
+Starting from Jan 2020, CDF covers a significant part of the infrastructure costs
+including our services and CI/CD instances running on Microsoft Azure.
+CDF provided us with legal assistance required to get code signing keys for the Jenkins project.
+Thanks to that, we were able to switch to a new link:https://github.com/jenkins-infra/release[Jenkins Release Infrastructure].
```

Comment:
```suggestion
This allowed the project to switch to a new, automated, link:https://github.com/jenkins-infra/release[Jenkins Release Infrastructure].
```

Reply:
The previous one is "fully automated" as well. i would rather reference Cloud, Infra as Code, etc. But not sure how to put it shortly

Reply:
```suggestion
This allowed the project to switch to a new, modern and fully automated, link:https://github.com/jenkins-infra/release[Jenkins Release Infrastructure].
```